src/components/markdown_editor/__snapshots__/markdown_editor.test.tsx.snap

```diff
@@ -219,7 +220,6 @@ exports[`OuiMarkdownEditor is rendered 1`] = `
         </button>
       </div>
       <input
-        autocomplete="off"
```

Comment:
Does removing this attribute change anything for the end user?

Reply:
This happens in [v12.0.6](https://github.com/react-dropzone/react-dropzone/releases/tag/v12.0.6), here is [PR](https://github.com/react-dropzone/react-dropzone/pull/1174), which explains the reasons for removing this attribute. In short, no, it should not affect the end user.